src/components/ReviewList.jsx

```diff
@@ -1,16 +1,44 @@
-import React from 'react';
+import React, {useState} from 'react';
 import Review from './Review.jsx';
+import FormModal from './FormModal.jsx';
+
+
 
 function ReviewList(props) {
-  const listReviews = props.example.map((review, i) =>
+const first = props.reviews[0];
+const second = props.reviews[1];
+
+  const [displayedReviews, setDisplayedReviews] = useState([first, second]);
+  const [counter, setCounter] = useState(2);
+
+  function addReviews() {
+     var nextReviews = props.reviews.slice(counter, counter + 2);
+     var combineReviews = displayedReviews.concat(nextReviews);
+     setDisplayedReviews(combineReviews);
+     setCounter(counter + 2);
+  };
+
+       //Modal===========
+       const [show, setShow] = useState(false);
+       const handleClose = () => setShow(false);
+       function handleShow() {setShow(true)};
+
```

Comment:
a bit confusing that these are tabbed in.  I like the commented out label, but keep em tabbed the normal amount